optimum/amd/brevitas/quantizer.py

```diff
@@ -0,0 +1,342 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
```

Comment:
Maybe a design choice, but could we rename the file to `quantization.py` to make it analogous to other optimum packages?